Userland/Libraries/LibCore/System.cpp

```diff
@@ -66,6 +66,71 @@ ErrorOr<void> beep()
     return {};
 }
 
+String Promises::to_string() const
+{
+    Vector<String> promises_builder;
```

Comment:
Why a Vector<String> and not a StringBuilder?

Reply:
StringBuilder can't remove the final space that would be inserted. Therefore, string join is cleaner.

Reply:
It can be a `Vector<StringView>` to avoid allocating strings